schemas/com.olark/offline_message/jsonschema/1-0-0

```diff
@@ -0,0 +1,122 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an Olark offline message event",
+  "self": {
+    "vendor": "com.olark",
+    "name": "offline_message",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "items": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "kind": {
+            "type": "string",
+            "maxLength": 64
+          },
+          "timestamp": {
```

Comment:
Do we know what format this timestamp is in?

Reply:
@alexanderdean That is also already done (but for some reason not marked as "outdated" in the PR

Reply:
Thanks